packages/endace/changelog.yml

```
@@ -0,0 +1,6 @@
+# newer versions go on top
+- version: "0.1.0"
+  changes:
+    - description: Initial draft of the package
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/10308
```

```suggestion
      link: https://github.com/elastic/integrations/pull/13423
```

change to this PR number